Sources/PackageCollectionsSigning/Key/ASN1/ASN1.swift

```diff
@@ -0,0 +1,389 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2021 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftCrypto open source project
```

Comment:
its unfortunate that we cannot use SwiftCrypto types here instead of re-defining them, is this something @Lukasa can help fix in SwiftCrypto?

Reply:
At this time the answer is no: the ASN.1 code is changing rapidly. While I do want to build out Swift ASN.1 infrastructure, until we get something approaching complete coverage of ASN.1 there's essentially no guarantees on stability (indeed, I plan to break it soon).

In the short term copy-pasting is fine. Longer term when we resolve the ASN.1 issue we should be taking an item to remove these files.

Reply:
Actually they can be removed sooner than then: as soon as SwiftPM can drop support for Catalina we no longer need these.